.gitignore

```diff
@@ -42,6 +42,9 @@ build
 manager-api
 
 output
+publish
```

Comment:
We use `publish` as a folder is not a good idea, can you provide some examples of other popular projects?

The act of ordering this line is to package the product, and we need to think about whether we need to do this here. 

If we need a package as the final product, then just packaged it in the build process.
If we use a package just to let users move it easily, let's call it `make package`, but I don't think it's needed and will take the confusion with `make build`.

cc @moonming @membphis any ideas?

Reply:
agreed. @gxthrj please refer to other repos first.
To be honest, I don’t know what PR does

Reply:
It is for the issue #787 
It seems that I didn't understood the meaning of issue.